package.json

```
@@ -711,6 +721,11 @@
           "when": "view == confluent-schemas && viewItem =~ /.*multiple-versions.*/",
           "group": "3_compare"
         },
+        {
+          "command": "confluent.topic.produce.fromFile",
+          "when": "view == confluent-topics && viewItem =~ /.*-topic.*/",
```

We'll need to make sure this action is only available when the user has the `WRITE` authorized operation
```suggestion
          "view == confluent-topics && viewItem =~ /.*-topic.*-authzWrite.*/",
```

---

And we'll need https://github.com/confluentinc/vscode/blob/main/src/models/topic.ts#L81 to include `WRITE` (until https://github.com/confluentinc/vscode/issues/659 is done, anyway)